master/buildbot/worker/base.py

```diff
@@ -296,7 +297,10 @@ def stopService(self):
 
     def startMissingTimer(self):
         if self.notify_on_missing and self.missing_timeout and self.parent:
-            self.stopMissingTimer()  # in case it's already running
+            try:
+                self.stopMissingTimer()  # in case it's already running
+            except error.AlreadyCalled:
```

Comment:
> Currently failing in worker start should be expected since we have no way of querying in which state the callLater instance is.

What do you mean? there is self.missing_timer.active(), and self.missing_timer is not None.
missing_timer is always reset to None when cancelled, and first thing after the callback is called.

There might be a small case when the timer has triggered, but its deferred is in the queue, but I really don't think this can happen. 
I dont think we should log something as there is nothing the user can do about that. This would lead to more confusion that anything,
So this is why I still prefer my suggestion.

Also, I would like to have a unit test to cover the situation.